modules/core/src/adapter/resources/framebuffer.ts

```diff
@@ -42,9 +42,9 @@ export abstract class Framebuffer extends Resource<FramebufferProps> {
   /** Height of all attachments in this framebuffer */
   height: number;
   /** Color attachments */
-  colorAttachments: TextureView[] = [];
+  colorAttachments: (TextureView | Texture)[] = [];
```

Comment:
The idea was to only store `TexturevViews` here in the class itself. For convenience we allow other types like `Texture` in the props, but then extract the default view. We can always get the texture from `view.texture`

If a Texture actually made it in here, we have a type hole somewhere.

Reply:
Looking again at the screenshot, I see that in this case it is a Renderbuffer. Support for renderbuffers is currently hacky in luma.gl and even changing the type to Texture doesn't correctly cover Renderbuffers.

I believe we agreed to move away from RenderBuffers and use depth textures instead.

Reply:
Good point, fixed here: https://github.com/visgl/deck.gl/pull/8567/commits/224d9af5f76845e68b42ef8915302793e0068bd2 

I noticed [that internally luma can still create a RenderBuffer](https://github.com/visgl/luma.gl/blob/master/modules/webgl/src/adapter/resources/webgl-framebuffer.ts#L88-L97), when the constructor is invoked with a string. I take it we want to move away from this also?